src/seedsigner/views/seed_views.py

```diff
@@ -323,20 +323,52 @@ def __init__(self):
 
 
     def run(self):
-        ret = self.run_screen(seed_screens.SeedAddPassphraseScreen, passphrase=self.seed.passphrase)
+        ret_passphrase, ret_btn = self.run_screen(seed_screens.SeedAddPassphraseScreen, passphrase=self.seed.passphrase)
 
-        if ret == RET_CODE__BACK_BUTTON:
-            return Destination(BackStackView)
-        
         # The new passphrase will be the return value; it might be empty.
-        self.seed.set_passphrase(ret)
-        if len(self.seed.passphrase) > 0:
+        self.seed.set_passphrase(ret_passphrase)
+        if ret_btn == RET_CODE__BACK_BUTTON:
+            if len(self.seed.passphrase) > 0:
+                return Destination(SeedAddPassphraseExitDialogView)
+            else:
+                return Destination(BackStackView)
+        elif len(self.seed.passphrase) > 0:
```

Comment:
And as if/else branches get nested or more complex, I tend to use whitespace to further emphasize the top levels:
```python
if blah:
    if foo:
        pass
    else:
        pass

elif blerg:
    pass

else:
    pass
```

Reply:
ACK as of https://github.com/SeedSigner/seedsigner/pull/563/commits/d8b7672cf48b93eb22abdf5796b420a1000e1af4